apps/music/js/metadata/album_art_cache.js

```diff
@@ -38,17 +96,22 @@ var AlbumArtCache = (function() {
 
     // See if we've already made a URL for this album. If so, return it.
     var cacheKey = makeCacheKey(fileinfo);
-    if (cacheKey && cacheKey in L1Cache) {
-      return Promise.resolve(L1Cache[cacheKey]);
+    if (cacheKey && cacheKey in L1Cache.thumbnail) {
```

Comment:
I just noticed that you're checking whether cacheKey exists here and skipping the L1 cache if so. But below you are apparently just assuming that cacheKey is defined.  So I think you can just remove the `cacheKey &&` here.